docs/reading_files.rst

```diff
@@ -268,3 +268,34 @@ You only need these details if you think caching may be causing problems.
 
 JSON was chosen as it can be easily inspected manually, and it's reasonably
 efficient to load the entire file.
+
+File Reading Issues
+-------------------
+
+Files at European XFEL storage migrate over time from GPFS to PNFS. The data 
+on PNFS usually always available for reading. But sometimes, reading may require
```

Comment:
```suggestion
on PNFS is usually always available for reading. But sometimes, this may require
```

"Data is" vs "Data are" is a matter of debate in English. If we want to avoid that altogether, we can say "Files... are"